charts/xui-webapp/values.preview.template.yaml

```diff
@@ -16,6 +16,8 @@ nodejs:
      HEALTH_WORK_ALLOCATION_TASK_API: http://wa-task-management-api-aat.service.core-compute-aat.internal/health
      HEALTH_ROLE_ASSIGNMENT_API: http://am-role-assignment-service-aat.service.core-compute-aat.internal/health
      HEALTH_CASE_CASEWORKER_API: http://rd-caseworker-ref-api-aat.service.core-compute-aat.internal/health
+     SERVICES_CCD_ACTIVITY_API_PATH: http://ccd-case-activity-api-pr-276.service.core-compute-preview.internal
```

Comment:
Preview URL - so likely to disappear at some point.

Reply:
Yes. This is purely for testing purposes. This should be removed prior to any merging, as discussed with @ernestman28. 😁